### DISCUSSION TITLE
1.21.0 update

### DISCUSSION THREAD

**Discussion by acaonweb** [Koenkk/zigbee2mqtt#8316] [General]
Sorry for my silly question, I have zzh! CC2652R Multiprotocol RF Stick flashed with then second last firmware
Upgrading to 1.21.0 is necessary to upgrade zzh! CC2652R Multiprotocol RF Stick firmware to the last?

thanx in advance 
Fabrizio

**Comment by Koenkk** [comment]
Z2M 1.21.0 will work fine with older CC2652 firmwares, no need to upgrade.

**Reply by acaonweb** [reply]
ok, thanks so  much
regards
Fabrizio
